core/state-machine/src/lib.rs

```diff
@@ -968,6 +989,26 @@ where
 	}
 }
 
+/// Reads child storage value from overlay or from the backend.
+fn try_child_read_overlay_value<H, B>(
```

Comment:
this is never used ?

Reply:
I am removing it (was use in may branch with child trie usage but is only an annoying warning here).